### ISSUE TITLE
Missing maps_projection in the checkpoint and batch_norm in the code

### ISSUE THREAD

**Issue by rishabhm12** [facebookresearch/VICRegL#13]
Hey guys,

There's no 'maps_projection' layer in **resnet50_alpha0p75** checkpoint, while the source code has it. Also, when l load the checkpoint, the global projection layer has batch norm layers but the code misses it.

Here's the diff:
**state dict from the checkpoint**
`module.projector.0.weight
module.projector.0.bias
module.projector.1.weight
module.projector.1.bias
module.projector.1.running_mean
module.projector.1.running_var
module.projector.1.num_batches_tracked
module.projector.3.weight
module.projector.3.bias
module.projector.4.weight
module.projector.4.bias
module.projector.4.running_mean
module.projector.4.running_var
module.projector.4.num_batches_tracked
module.projector.6.weight`

**state dict from the repo's global and local projection layer:**
`module.maps_projector.0.weight
module.maps_projector.0.bias
module.maps_projector.1.weight
module.maps_projector.1.bias
module.maps_projector.3.weight
module.maps_projector.3.bias
module.maps_projector.4.weight
module.maps_projector.4.bias
module.maps_projector.6.weight
module.projector.0.weight
module.projector.0.bias
module.projector.1.weight
module.projector.1.bias
module.projector.3.weight
module.projector.3.bias
module.projector.4.weight
module.projector.4.bias
module.projector.6.weight`

Please share the complete checkpoint, if possible

Thank you!

**Comment by rishabhm12** [comment]
Looks like VICReg checkpoint has been uploaded instead of VICRegL

**Comment by Adrien987k** [comment]
Hi,

Thanks for pointing out the issue. I will investigate and upload the correct model as soon as possible.

**Comment by rishabhm12** [comment]
Hi Adrien,
Do comment on this thread after you have uploaded the right checkpoint.
Thank you!

**Comment by Adrien987k** [comment]
Hi,

The correct checkpoints have been uploaded. The resnet-50 models were trained with batch_norm in the projectors and not layer_norm, the code has been fixed accordingly.

Let me know if you still have any issue.

**Comment by rishabhm12** [comment]
Hi Adrien,
I validated the checkpoint, it's correct seems now. Thanks a ton :)
I am closing the issue.
